### ISSUE TITLE
[Bug]: FreeTube starts, but window is no reachable nor visible

### ISSUE THREAD

**Issue by not-AMM** [FreeTubeApp/FreeTube#2502]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for a bug report that matches the one I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.

### Describe the bug

I updated FreeTube from 0.17.0 to 0.17.1. I didn't use it during 0.17.0, so I don't know if it worked by then. 

After this, trying to open FreeTube only showed a program running in the taskbar, but the window is not reachable nor visible. I have to close it from the taskbar. I could only find the errors by opening from terminal. 

### Expected Behavior

FreeTube window is visible and usable.

### Issue Labels

usability issue

### FreeTube Version

v0.17.1

### Operating System Version

Windows 10 Version 21H2

### Installation Method

.exe

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

Before 0.17.0 (I think)

### Additional Information

![imagen](https://user-images.githubusercontent.com/43765089/185768957-34f7a6c7-8b43-442e-841a-4092551e0e9e.png)

```
TypeError: Cannot destructure property 'value' of 'object null' as it is null.
    at C:\Users\usr\AppData\Local\Programs\FreeTube\resources\app.asar\dist\main.js:1:4801
    at async W (C:\Users\usr\AppData\Local\Programs\FreeTube\resources\app.asar\dist\main.js:1:4757)
    at async App.<anonymous> (C:\Users\usr\AppData\Local\Programs\FreeTube\resources\app.asar\dist\main.js:1:9167)
(node:1628) UnhandledPromiseRejectionWarning: Error: Cannot create a secure cookie from an insecure URL
(Use `FreeTube --trace-warnings ...` to show where the warning was created)
(node:1628) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:1628) UnhandledPromiseRejectionWarning: Error: Cannot create a secure cookie from an insecure URL
(node:1628) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
[6316:0820/175855.740:ERROR:gpu_init.cc(457)] Passthrough is not supported, GL is disabled, ANGLE is
```

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Is this still an issue on [the latest nightly build](https://docs.freetubeapp.io/development/nightly-builds)?

**Comment by MaxiMii05** [comment]
This is the issue I am always encountering on Wayland.

**Comment by MaxiMii05** [comment]
Can confirm that this issue is present in the newest nightly AppImage, however in contrast to the Flatpak I'm using, the window doesn't stay blank. Still, the errors are present in the journal.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Just to clarify what build did u use?

**Comment by MaxiMii05** [comment]
I was using [nightly-2164](https://github.com/FreeTubeApp/FreeTube/actions/runs/3287208918).
The blank window in the Flatpak version might be unrelated to this issue. Also the errors are visible in the journal for the nightly build, but I'm not sure what is 'broken' because of that. Playing a video works like normal, but I didn't do any further testing.
Also, `[6316:0820/175855.740:ERROR:gpu_init.cc(457)] Passthrough is not supported, GL is disabled, ANGLE is` is not present in my testing.
I'm not sure what to make out of this. Maybe we're dealing with three different issues, maybe some of them were already fixed. Let me know if I can help in any way. For me, the Flatpak version currently not working with Wayland (blank window) is the reason I'm not using Wayland yet.

- Fedora 36
- Nvidia GPU with proprietary drivers version 520

**Comment by niboe-boop** [comment]
I have just come across this myself and I'd like to add to this. I have the same symptom, the window does not appear at all. See this screenshot of my alt+tab.
![alt tab](https://user-images.githubusercontent.com/76244859/213894904-a5c27c88-8329-4f88-819f-954763da90c8.png)

I'm running the exact same version of Windows as OP and using [nightly build 2568](https://github.com/FreeTubeApp/FreeTube/actions/runs/3974207060). Once the issue appeared, it now persists across all fresh installations of Freetube that I've tried. For me, it began when I left a video paused in fullscreen for a long time and was using other applications in other windows. Then I left my computer on overnight, closing laptop lid, and when I tried to return to the video in the morning is when I noticed.

The error message I get is slightly different:
![error](https://user-images.githubusercontent.com/76244859/213895020-7cd8f5ae-c5ed-4e06-b08a-d127d9c9efbb.png)

Note: Although the nightly build suffers from this bug, whatever changed my OS to become such that it triggers this bug happened while running v0.18.0. I can't try to replicate that change on the nightly build until I can undo whatever this is on the OS side.
